config/locales/en.yml

```diff
@@ -21,12 +21,14 @@ en:
     has_many_new: "Add New %{model}"
     has_many_delete: "Delete"
     has_many_remove: "Remove"
+    move: "MOVE"
```

Comment:
I guess we should keep the case consistent. I'm up for making this lowercase.

Reply:
I agree, I added it uppercase because in the source is uppercase

Reply:
This will change in #3862 but I agree we should have localized. That way I can use it in #3862 as alt text. Agree with @deivid-rodriguez on keeping case consistent. The text in the source will be removed anyway. Thanks!